diff -durN tuxtype-1.0.orig/tuxtype/globals.h tuxtype-1.0/tuxtype/globals.h
--- tuxtype-1.0.orig/tuxtype/globals.h	Wed Aug  1 13:22:00 2001
+++ tuxtype-1.0/tuxtype/globals.h	Tue Aug  7 14:41:06 2001
@@ -92,7 +92,7 @@
 
 /* paths */
 #define FNLEN 	200
-#define PATHNUM 6
+#define PATHNUM 5
 
 //Game difficulty levels
 #define EASY				0
@@ -135,4 +135,4 @@
 
 #define MUSIC_FADE_OUT_MS	80
 #define FADE_IN_TITLE_MUSIC 1000
-#endif // __GLOBALS_H__
\ No newline at end of file
+#endif // __GLOBALS_H__
diff -durN tuxtype-1.0.orig/tuxtype/graphics.h tuxtype-1.0/tuxtype/graphics.h
--- tuxtype-1.0.orig/tuxtype/graphics.h	Wed Aug  1 13:22:00 2001
+++ tuxtype-1.0/tuxtype/graphics.h	Tue Aug  7 14:40:24 2001
@@ -26,7 +26,7 @@
 int menu_item[TITLE_MENU_ITEMS + 1][TITLE_MENU_DEPTH + 1];
 
 const char PATH[PATHNUM][FNLEN] =
-    { ".", "data", "/usr/local/share/tuxtype", "/usr/share/tuxtype",
+    { ".", "data", PKGDATADIR,
 	"../data", "/usr/share/pixmaps/tuxtype" };
 	
 #endif //  __GRAPHICS_H__
diff -durN tuxtype-1.0.orig/tuxtype/playgame.h tuxtype-1.0/tuxtype/playgame.h
--- tuxtype-1.0.orig/tuxtype/playgame.h	Wed Aug  1 13:22:00 2001
+++ tuxtype-1.0/tuxtype/playgame.h	Tue Aug  7 14:40:58 2001
@@ -205,7 +205,7 @@
 #define FISH4_MAX_Y		10
 
 const char WORD_PATH[PATHNUM][FNLEN] =
-    { ".", "data", "/usr/local/share/tuxtype", "/usr/share/tuxtype",
+    { ".", "data", PKGDATADIR,
 	"../data", "/usr/share/pixmaps/tuxtype" };
 
 #endif				//KCASCADE. FOURTH FRAMEsplay this object6s --*/ __PLAYGAME_H__
