diff -urN tuxtype-1.0.1.orig/tuxtype/globals.h tuxtype-1.0.1/tuxtype/globals.h
--- tuxtype-1.0.1.orig/tuxtype/globals.h	Sun Aug  4 12:28:56 2002
+++ tuxtype-1.0.1/tuxtype/globals.h	Sun Aug  4 12:29:59 2002
@@ -92,7 +92,7 @@
 
 /* paths */
 #define FNLEN 	200
-#define PATHNUM 3
+#define PATHNUM 6
 
 //Game difficulty levels
 #define EASY				0
@@ -135,4 +135,5 @@
 
 #define MUSIC_FADE_OUT_MS	80
 #define FADE_IN_TITLE_MUSIC 1000
-#endif // __GLOBALS_H__
\ No newline at end of file
+#endif // __GLOBALS_H__
+
